homework-14/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework-14</title>
+</head>
+
+<body>
+    <div class="container">
+        <section class="section1">
+            <label for="year-to-centure">

```

Comment:
```suggestion
            <label for="input1">
```
The value of the for attribute must be a single **id** for a labelable form-related element in the same document as the <label> element.